CMakeLists.txt

```diff
@@ -43,6 +43,31 @@ if (NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE Release CACHE STRING "choose 'Release' as default build type" FORCE)
 endif ()
 
+if(NOT CMAKE_CXX_COMPILER)
+    set(CMAKE_CXX_COMPILER g++-7)
+endif()
+
+# Fill backends with inference software directory
+if(DEFINED TENSORRT_DIR)
+    list(APPEND MMDEPLOY_TARGET_BACKENDS trt)
+endif()
+
+if(DEFINED ONNXRUNTIME_DIR)
+    list(APPEND MMDEPLOY_TARGET_BACKENDS ort)
+endif()
+
+if(DEFINED pplnn_DIR)
+    list(APPEND MMDEPLOY_TARGET_BACKENDS pplnn)
+endif()
+
+if(DEFINED ncnn_DIR)
+    list(APPEND MMDEPLOY_TARGET_BACKENDS ncnn)
+endif()
+
+if(DEFINED Torch_DIR)
+    list(APPEND MMDEPLOY_TARGET_BACKENDS torchscript)
+endif()
+
```

Comment:
Maybe need to add `InferenceEngine_DIR` for `OpenVino`?
The same as zh-cn version.

Reply:
done.